cristherose/index.html

```diff
@@ -0,0 +1,23 @@
+
+<!DOCTYPE html>
+<html lang="pt-BR">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>A historia de uma peixe</title>
+</head>
+<body>
+    <a href=" https://cristherose.github.io/turma-8-2021-03-30/."></a>
```

Comment:
```suggestion
    <a href=" https://cristherose.github.io/turma-8-2021-03-30/.">texto do link aqui</a>
```
adicionar texto para o link, verificar se o link está correto